ecosystem/sep-0006.md

```diff
@@ -475,7 +482,8 @@ The response should be a JSON object like:
               "dest": {"description": "your bank account number" },
               "dest_extra": { "description": "your routing number" },
               "bank_branch": { "description": "address of your bank branch" },
-              "phone_number": { "description": "your phone number in case there's an issue" }
+              "phone_number": { "description": "your phone number in case there's an issue" },
+              "country_code": { "description": "The ISO 3166-1 alpha-3 code of the user's current address"}
```

Comment:
❓ Is this the country code of the users address or the bank?

Reply:
User, as defined in the `description` and request parameter list.

Reply:
Got it. Why does it appear under the bank account field for withdrawal but not for deposit?

Reply:
Thats just the way `/info` is defined, assets under the `withdraw` section specify a `types` key, and each defined type has it's own set of `fields`. Assets under the `deposit` section just have a single `fields` object.